Main.java

```diff
@@ -9,7 +10,21 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    
+    //create a Scanner for user input
+   Scanner input = new Scanner(System.in);
+    
+
+    // ask user for temp
+    System.out.println("Please enter a temperature you would like to convert in degrees Celsius.");
+
+    // variable Celsius is the temp
+    int C = input.nextInt();
+
+  // Declare and calculate the temp
+  int F=(C*9)/5 +32;
+
+  //displayes the conversion
+System.out.println(C + " is the same as " + F);

```

Comment:
The reason the test cases are failing is because this output is not quite the same as the example shown. IF it had matched, I believe it would have passed